cspell.json

```diff
@@ -0,0 +1,99 @@
+{
+    "dictionaries": ["html", "css", "javascript"],
+    "words": [
+        "assistive",
+        "Scrollable",
+        "treegrid",
+        "describedby",
+        "valuetext",
+        "Menubutton",
+        "focusable",
+        "Listbox",
+        "tabindex",
+        "Treeview",
+        "IDREF",
+        "activedescendant",
+        "Listboxes",
+        "Contentinfo",
+        "tablist",
+        "portlet",
+        "NVDA",
+        "gridcell",
+        "haspopup",
+        "Cutey's",
+        "setsize",
+        "posinset",
+        "Rearrangeable",
+        "Focusability",
+        "valuemin",
+        "valuenow",
+        "tabpanel",
+        "WCAG",
+        "Checkboxes",
+        "alertdialog",
+        "columnheader",
+        "Tatiana",
+        "colcount",
+        "valuemax",
+        "listitem",
+        "transactinide",
+        "transuranic",
+        "visua",
+        "Capitan",
+        "thegoogle",
+        "Dmuddy",
+        "multithumb",
+        "Dubnium",
+        "Transuranium",
+        "Seaborgium",
+        "Dubnium",
+        "Minard",
+        "repo",
+        "xlink",
+        "Cook'n",
+        "Treegrids",
+        "treeitem",
+        "Combobox",
+        "rowgroup",
+        "Paciello",
+        "Deque",
+        "docx",
+        "Anjou",
+        "imgs",
+        "submenus",
+        "submenu",
+        "colindex",
+        "rowindex",
+        "comboboxes",
+        "radiogroup",
+        "Bookmarklets",
+        "sidepanel",
+        "signup",
+        "Bookmarklet",
+        "rowheader",
+        "colheader",
+        "Seckel",
+        "Bosc",
+        "depricated",
+        "kbdshortcuts",
+        "keyshortcuts",
+        "IDREFS",
+        "Bohrium",
+        "Hassium",
+        "Minard's",
+        "consectetur",
+        "amet",
+        "Frahm",
+        "Obel",
+        "walkability",
+        "rowcount",
+        "labelledby"
+    ],
+    "ignoreRegExpList": [
+        "id=\"(?:[^\\\"]+|\\.)*\"",
```

Comment:
@jnurthen excluded a few of the IDREF attributes in here, not sure if any others should be added